.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Support request
```

Comment:
@36degrees Do you think this would be better if it was less specific about getting 'support'? We could do something like this to mirror what the design system page says:

```
name: Get in touch
url: https://design-system.service.gov.uk/get-in-touch/
about: If you’ve got a question, idea or suggestion get in touch with the Design System team.
```

Reply:
Yep, although raising an issue is also a valid way of 'getting in touch' and we don't want to mis-direct people who would otherwise make a suggestion via an issue so… what about something like…

```
name: Get in touch another way
url: https://design-system.service.gov.uk/get-in-touch/
about: Find out how to get in touch via email or Slack
```